site/content/en/_index.html

```diff
@@ -7,83 +7,99 @@
 - src: "start.jpg"
   title: "start"
 ---
-{{< blocks/cover title="Welcome to minikube" image_anchor="top" height="full" color="orange" >}}
-<div class="mx-auto">
-	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
-		Read the docs! <i class="fas fa-arrow-alt-circle-right ml-2"></i>
-	</a>
-	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/google/docsy-example">
-		Download <i class="fab fa-github ml-2 "></i>
-	</a>
-	<p class="lead mt-5">The local Kubernetes environment for macOS, Linux, and Windows.</p>
-	<div class="mx-auto mt-5">
-		{{< blocks/link-down color="info" >}}
+
+<!-- Welcome to minikube -->
+<section class="row td-box td-box--white position-relative td-box--gradient td-box--height-auto">
+	<div class="container td-arrow-down">
+		<div class="row">
+			<div class="col-sm-8">
+				<h1>Welcome to minikube!</h1>
+				<p class="lead mt-5" style="text: nowrap">Local Kubernetes, focused on application development &amp; education</p>
+				<a class="btn btn-lg btn-primary mr-3 mb-4" href="/docs/">
+					Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+				</a>
+				<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/kubernetes/minikube">
+					View Repository <i class="fab fa-github ml-2 "></i>
+				</a>
+			</div>
+			<div class="col">
+				{{< imgproc "logo" Fit "220x220" >}}{{< /imgproc >}}
+			</div>
+		</div>
 	</div>
-</div>
-{{< /blocks/cover >}}
+</section>
 
 
-{{% blocks/lead color="primary" %}}
+<!-- Sales pitch -->
+{{< blocks/lead >}}
 
-minikube is a local Kubernetes environment focused on application developers.
+<div class="container">
+<h2>Instantly productive.</h2>
+
+A single command away from reproducing your production environment, from the comfort of localhost.
 
 {{< imgproc "start" Fit "730x239" >}}{{< /imgproc >}}
 
-Kubernetes certified, mother approved.
-{{% /blocks/lead %}}
+<h3>Highlights</h3>
+<ul class="fa-ul">
+<li><i class="fas fa-check"></i> Always supports the latest Kubernetes release, as well as previous versions</li>
+<li><i class="fas fa-check"></i> Cross-platform (Linux, macOS, Windows)</li>
+<li><i class="fas fa-check"></i> Provides a Docker API endpoint</li>
```

Comment:
I'm not sure if this is a bug or a feature, but it sure seems popular.

Reply:
> I'm not sure if this is a bug or a feature (the Docker API), but it sure seems popular.

A feature, but agreed. I suspect 2 years from know tools will have evolved to where it isn't an important feature.



> We need to talk about VirtualBox some time.

This PR doesn't mention VirtualBox, but agreed. That said, at the moment, it seems that VirtualBox is still the easiest environment to use on all platforms.

> The Docker API feature is a similar story, we have both "Reusable Docker daemon" and "Any container runtime". It would be awesome if we could focus on use cases, like building images or providing a CRI.

Agreed. If you have any suggestions on how to recategorize the features to improve this, I'm all ears.

> I would rather talk about different standards like the OCI, and how it supports your Dockerfile and your regular Kubernetes commands. Not so much about which vendor implementation that we are providing ?

In this section I was trying to highlight ways in which our tool was different than other  similar tools. I've gone ahead and mentioned the multiple runtimes here though.